snap/snapcraft.yaml

```diff
@@ -51,19 +53,20 @@ parts:
         - wheel
         - pip
         - setuptools
-    requirements:
-        - requirements-focal.txt
+    python-requirements:
+        - requirements-jammy.txt
         - requirements.txt
+    build-attributes:
+        - enable-patchelf
     build-environment:
-        - "CFLAGS": "$(pkg-config python-3.8 yaml-0.1 --cflags)"
+        - "CFLAGS": "$(pkg-config python-3.10 yaml-0.1 --cflags)"
     after: [rockcraft-libs]
     override-build: |
       snapcraftctl build
       # python3 fixup symlink (snapcraft bug)
-      ln -sf ../usr/bin/python3.8 $SNAPCRAFT_PART_INSTALL/bin/python3
-        mkdir -p $SNAPCRAFT_PART_INSTALL/libexec/rockcraft
-        mv $SNAPCRAFT_PART_INSTALL/bin/craftctl $SNAPCRAFT_PART_INSTALL/libexec/rockcraft/
-        sed -i -e '1 s|^#!/.*|#!/snap/rockcraft/current/bin/python|' $SNAPCRAFT_PART_INSTALL/libexec/rockcraft/craftctl
+      mkdir -p $SNAPCRAFT_PART_INSTALL/libexec/rockcraft
```

Comment:
can we use `CRAFT_PART_INSTALL` instead?? You know, in a "dogfooding" sense

Reply:
you are correct!